service/service.go

```diff
@@ -1092,3 +1092,22 @@ func (s *service) validateIsiPath(ctx context.Context, volName string) (string,
 
 	return isiPath, nil
 }
+
+func (s *service) GetIsiPathByName(ctx context.Context, volName string) (string, error) {
+	if s.k8sclient == nil {
+		return "", fmt.Errorf("no k8s clientset")
```

Comment:
This does not seem like a useful error for an end user.

Reply:
I've followed the same format for error messages as used in the existing code.  We can track it as part of separate PR